nbdime/merging/autoresolve.py

```diff
@@ -190,6 +189,8 @@ def strategy2action_dict(resolved_base, le, re, strategy, path, dec):
         mval = max(bval, lval, rval)
         if bval == mval:
             return []
+        elif lval == mval == rval:
+            dec.action = "either"
```

Comment:
The presentation framework can use it, as it should be of interest to the user whether local work picked, or the changes agree.